src/enable_variable_ops/ngraph_remove_ngraphassigns.cc

```diff
@@ -0,0 +1,97 @@
+/*******************************************************************************
+ * Copyright 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "ngraph_remove_ngraphassigns.h"
```

Comment:
Need to add the full path. See @kanvi-nervana's PR.

Reply:
Will it work as her PR is not merged? I think it renames the src directory to ngraph-bridge directory. I was thinking of adding the full Paths and testing once her PR is in.